src/vfs2.c

```diff
@@ -38,6 +38,9 @@
 
 #define READ_MARK_UNUSED 0xffffffff
 
+#define DB_HEADER_SIZE 100
+#define DB_HEADER_NPAGES_OFFSET 28
```

Comment:
Extremely nit: but what about `DB_FILE_HEADER_SIZE` and `DB_FILE_HEADER_NPAGES_OFFSET`, I was not sure what `DB` was referring to without reading the usage.

Reply:
Yes, this should have `DB_FILE` or `MAIN_FILE` somewhere.